react/features/recording/components/LiveStream/AbstractLiveStreamButton.js

```diff
@@ -119,14 +122,18 @@ export function _mapStateToProps(state: Object, ownProps: Props) {
         } = state['features/base/config'];
         const { features = {} } = getLocalParticipant(state);
 
-        visible = liveStreamingEnabled
-            && (!enableFeaturesBasedOnToken
-                || String(features.livestreaming) === 'true');
+        visible = liveStreamingEnabled;
+
+        if (enableFeaturesBasedOnToken) {
```

Comment:
Should this check also check for liveStreamingEnabled being truthy? For example, if liveStreamingEnabled is false, then should the button always be hidden?

Reply:
Yes, if it is not enabled there is no point of checking that. 
So we can consider these settings as on global for the whole deployment and a second set which is per user.